create-cert-bundle.py

```diff
@@ -0,0 +1,46 @@
+import os
+import logging
+import sys
+
+logger = logging.getLogger()
+handler = logging.StreamHandler(sys.stdout)
+logger.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+if os.environ.get("HTTPS_PROXY") is None:
+    exit()
+logger.info("Creating certificate bundle with proxy root CA, to be used by pip")
+try:
+    import certifi
+
+except ImportError:
+    try:
+        from pip._vendor import certifi
+    except Exception as e:
+        logger.exception("Error creating certificate bundle with proxy root CA")
+        exit()
+try:
+    p1 = certifi.where()
+    p2 = os.environ.get("GG_ROOT_CA_PATH")
+    p3 = './ca-bundle.crt'
+    f1 = open(p1, 'r')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code contains a resource that might not be closed properly. This can cause a resource leak that slows down or crashes your system.

**Fix**
Consider closing the following resource: *f1*. The resource is allocated by call *builtins.open*. The resource is referenced at lines 29, 31, 34. The resource is closed at line 34. There are other execution paths that do not contain closure statements, for example, when *TextIOWrapper.write()* throws an exception. To prevent this resource leak, close *f1* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).